plugins/obs-filters/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-project(obs-filters)
+﻿project(obs-filters)
```

Comment:
Supposedly it's fixed in the latest VS2017 update as of 20 hours ago?

https://developercommunity.visualstudio.com/content/problem/21744/vs2017-rc-breaks-the-encoding-of-my-files.html#reply-93979